.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: Windows Rtools44
+name: Rtools & ARM64 CI
```

Comment:
I think it's worth adding a comment somewhere (might as well be up here at the top) explaining the historical reasons for why this file is using pwsh -- I think it's fine to do so, but it will certainly look odd to someone in the future if the context is lost